enginetest/trigger_queries.go

```diff
@@ -2081,6 +2081,647 @@ end;`,
 			},
 		},
 	},
+	// Insert Queries that fail, test trigger reverts
```

Comment:
These tests are good, but they all need to have a version that inserts / updates multiple rows in a single statement, and the error happens on 2nd row. If that's not possible yet that's fine, let's move in the direction. But document what works and what doesn't so far with skipped tests.

Reply:
Since I'm dropping Transactions when INSERT fails, inserting multiple rows works as expected. However, testing with the memory harness instead of a dolt harness does not work.